orc-rt/docs/Building-ORC-RT.rst

```diff
@@ -0,0 +1,90 @@
+.. _Building_ORC_RT:
```

Comment:
nit: We might want to consider using Markdown instead of RST for the docs in a new project. Sphinx supports Markdown, and a bunch of the newer projects are using Markdown exclusively while we keep piecemeal transitioning existing projects.